lib/twap/events/self_interval_tick.js

```diff
@@ -42,9 +44,15 @@ const onSelfIntervalTick = async (instance = {}) => {
     return
   }
 
+  if (!isMarketOrder && !isBookUpdated) {
+    scheduleTick()
+    return
+  }
+
   if (!tradeBeyondEnd && !_isEmpty(orders)) {
     const [, t] = timeout(cancelDelay)
     await t()
+    await updateState(instance, { isBookUpdated: false })
```

Comment:
this is on cancel... but after sending an order, shouldn't the state change to `{ isBookUpdated: false }` too?

Reply:
will add this!!